File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/PokemonAllMovePpRestoreModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class PokemonAllMovePpRestoreModifierItem  extends PokemonModifierItem {

    public static final String TARGET = "PokemonAllMovePpRestoreModifierType";

    private int restorePoints;

}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/PokemonAllMovePpRestoreModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

class PokemonAllMovePpRestoreModifierItem : PokemonModifierItem() {

    companion object {
        const val TARGET = "PokemonAllMovePpRestoreModifierType"
    }

    var restorePoints: Int = 0
}